SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -813,6 +819,31 @@ void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessT
 
 	if (bIsServer)
 	{
+		// Check if this is a dormant entity, and if so retire the entity
+		if (PackageMap != nullptr)
+		{
+			const Worker_EntityId EntityId = PackageMap->GetEntityIdFromObject(ThisActor);
+
+			// If the actor is an initially dormant startup actor that has not been replicated.
+			if (EntityId == SpatialConstants::INVALID_ENTITY_ID && ThisActor->IsNetStartupActor())
+			{
+				UE_LOG(LogSpatialOSNetDriver, Log, TEXT("Creating a tombstone entity for initially dormant statup actor. "
+					"Actor: %s."), *ThisActor->GetName());
+				Sender->CreateTombstoneEntity(ThisActor);
+				return;
```

Comment:
Should we be returning here? What about the logic after the `bIsServer` block?

Reply:
I think you're right. I reordered this at some point and it looks like it should return. Will put an else after the block and then it let it do it's stuff.

Reply:
It's only the specific block underneath it (stuff that depends on having an entity ID that I wanted to early out on).